tests/accounts/test_views.py

```diff
@@ -1,20 +1,37 @@
 import calendar
 import datetime
+import sys
+from importlib import reload
 from urllib.parse import quote
 
+from django.conf import settings
 from django.contrib.auth import get_user_model
-from django.test import Client, TestCase
-from django.urls import reverse
+from django.test import Client, TestCase, override_settings
+from django.urls import clear_url_caches, reverse
 from django.utils import timezone
 from oauth2_provider.models import get_access_token_model, get_application_model
 
 from accounts.models import User
 from accounts.serializers import UserSearchSerializer, UserSerializer
 
 
+def reload_urlconf():
+    """
+    reloads the urlconfs after settings variables are updated
+    """
+    urlconf = settings.ROOT_URLCONF
+    acc_urls = "accounts.urls"
+    if urlconf in sys.modules and acc_urls in sys.modules:
+        clear_url_caches()
+        reload(sys.modules[urlconf])
+        reload(sys.modules[acc_urls])
+
+
+@override_settings(IS_DEV_LOGIN=False)
```

Comment:
Once we flip the default for IS_DEV_LOGIN to be false, we can remove the added lines here